src/components/Auth/useAuthProvider.ts

```diff
@@ -62,7 +62,9 @@ export const useAuthProvider = () => {
         logout()
         throw new Error('No bearer token')
       }
-      headers.append('Authorization', `Bearer ${bearer}`)
+      if (bearer) {
```

Comment:
Why is this check needed while above `if (!bearer)` is also done?

Reply:
You are right... I was trying to add support so we could use `bearedFetch` for both public and private calls, but obviously it would not work, since it's throwing an error before...

But that logic doesn't make much sense anyway... if there's no token, why logout? We're only storing the bearer as session information, so if you get into that line, there's no need to logout...

I personally would remove that first check, and leave it open to be used for both public and private calls. It already handles login-out users in case of error, so right now... we would have an app crash for all these calls (if we logout the user, the next `bearedFetch` call would fail with the `No bearer token` error).

Reply:
LGTM